corehq/ex-submodules/fluff/pillow.py

```diff
@@ -20,8 +21,8 @@ def __init__(self, indicator_class, delete_filtered=False):
         self.document_class = indicator_class.document_class
         self.save_direct_to_sql = indicator_class().save_direct_to_sql
         self.deleted_types = indicator_class.deleted_types
-        self.doc_type = indicator_class.document_class._doc_type
         self.wrapper = indicator_class.wrapper or indicator_class.document_class
```

Comment:
I did a quick grep and didn't see anything in custom pillows that ever used it. I'm guessing htat something old used it and it was never cleaned up